test/Makefile

```diff
@@ -254,10 +327,17 @@ test-reversible: plugin test_write_plugin
 # of test_write which prints them in the header output. The values
 # here are for accuracy mode and tolerance of 0.001.
 # 
-# A bug-fix patch to h5repack_parse.c is required for this test.
+# A bug-fix patch to h5repack_parse.c is required for this test on
+# older HDF5 versions (like <= 1.8.10).
 #
 test-h5repack: plugin mesh.h5 patch
-	@env LD_LIBRARY_PATH=$(HDF5_LIB):$(LD_LIBRARY_PATH) HDF5_PLUGIN_PATH=$(H5Z_ZFP_PLUGIN) $(HDF5_BIN)/h5repack -f UD=32013,6,3,0,3539053052,1062232653,0,0 \
+	@echo " "; \
+	pad=$$(printf '%*s' "$(padlimit)"); \
+	pad=$${pad// /.}; \
+	x="h5repack -f UD=1,32013,6,3,0,3539053052,1062232653,0,0"; \
```

Comment:
If I understand the help of h5repack application correctly, it should be specified as
```
h5repack -f UD=32013,0,6,3,0,3539053052,1062232653,0,0
```
At least, if we want to make use of the H5Z-ZFP filter during the repacking of the HDF5 file. However, I use this line I got the HDF5 error
```
HDF5-DIAG: Error detected in HDF5 (1.12.1) thread 0:
  #000: /glb/data/SeismicApps3/nljbl6/HDF5.custom/1.12.1_patch/source/src/H5T.c line 2090 in H5Tget_class(): not a datatype
    major: Invalid arguments to routine
    minor: Inappropriate type
```
Currently, I have no idea what the reason for this error is.

Note that in the original specification we made accidentally use of the GZIP filter.

Reply:
Correct, the filter ID should be first. Are you using a version of h5repack that has HDF5 statically linked in? HDF5 using dynamic plugins use the dlopen call, which must use shared libraries. Could you see if you have _h5repack-shared_?

Reply:
Ok, I think I know what I did...I used default paths to `h5repack` and am getting `1.10.0-patch` as the version. I think that still required the leading `1` or `0` to indicate mandatory or optional filter. Let me go check which version our CI is using.